File: src/web3/python/handshake.py
import time
import hashlib

class BitcoinHandshake:
    @staticmethod
    def create_handshake_payload(message="neurotoken-handshake"):
        hashed = hashlib.sha256(message.encode()).hexdigest()
        return {
            "version": "1.0",
            "algorithm": "sha256",
            "payload": hashed,
            "timestamp": time.time()
        }

File: src/web3/js/handshake.js
import crypto from "crypto";

export class BitcoinHandshake {
  static createHandshakePayload(message = "neurotoken-handshake") {
    const hash = crypto.createHash("sha256")
      .update(message)
      .digest("hex");

    return {
      version: "1.0",
      algorithm: "sha256",
      payload: hash,
      timestamp: Date.now()
    };
  }
}
